### PR TITLE
436 update readme and documentation

### LINKED ISSUE #436: Documentation: Wiki created, updating readme and other docs.
### Describe the Current Behavior/Feature:
Updating README to reflect changes to wiki

---
### Proposed Behavior/Feature:
remove some duplicated content and reference wiki

---
### Rationale:
reduce redundancy of documentation

### PR DESCRIPTION
**Pull request by Jared-Krajewski**
## This PR:
Resolves #436 
Wiki was added and some documentation has been placed in there along with a few updates since so much has changed. 

- main README.md and RESOURCES.md trimmed of most duplicate information.
- deleted CONTRIBUTING.md: [link to info in wiki](https://github.com/codeforpdx/PASS/wiki/Development#contribution-guidelines)
- deleted ZXing_barcode: [link to info in wiki](https://github.com/codeforpdx/PASS/wiki/Resources#APIs-and-Libraries-used-by-PASS)
- delete ROADMAP.md: [link to info in wiki](https://github.com/codeforpdx/PASS/wiki/Introduction#overview-of-pass)

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,6 +1,6 @@
 # PASS - Personal Access System for Services
```
Comment (leekahung) [inline README.md:2]: Could we have the images be placed within this repo instead?

Comment (leekahung) [reply README.md:2]: Oops, wrong line. Was meant to be Line 3 with the assets.

Comment (Jared-Krajewski) [reply README.md:2]: I guess I forgot to add it to the wiki assets when I was figuring out the dark/light mode for the wiki. Does sourcing from the PASS wiki work for you? If not I can add the light/dark mode to the assets folder of PASS itself and source from there assuming the in app logo will get updated. Either way the same image is used in the wiki and PASS.

Comment (leekahung) [reply README.md:2]: Yeah, I think that should be fine.

**Review by leekahung — APPROVED** [review]
Looks good to go.

**Pull request merged**